components/brave_news/browser/feed_v2_knobs.h

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_NEWS_BROWSER_FEED_V2_KNOBS_H_
+#define BRAVE_COMPONENTS_BRAVE_NEWS_BROWSER_FEED_V2_KNOBS_H_
+
+namespace brave_news::knobs {
+namespace switches {
+
+// An integer indicating the minimum number of cards (following the hero) in a
+// block.
+extern const char kBraveNewsMinBlockCards[];
```

Comment:
@LorenzoMinto are there any other knobs you think it'd be a good idea to expose here?